packages/ui/ImageItem/tests/ImageItem-specs.js

```diff
@@ -35,12 +35,12 @@ describe('ImageItem', () => {
 		expect(actual).toHaveLength(0);
 	});
 
-	test('should use a `Row` when `orientation="horizontal"`', () => {
+	test('should use a `div` when `orientation="horizontal"`', () => {
 		const subject = shallow(
 			<ImageItemBase orientation="horizontal" />
 		);
 
-		const actual = subject.find('Row.imageItem');
+		const actual = subject.find('div.imageItem');
```

Comment:
I think that the we might use something else except `'div.imageItem'` because whatever value for `orientation` has `horizontal` or `vertical`, there is the `div.imageitem`.